lib/tf-lock/tf-lock-release

```diff
@@ -103,8 +100,12 @@ def main():
 
     from os import environ
 
+    summaries = []
     for path in paths:
-        tf_lock_release(path, env=environ.copy())
+        summary = tf_lock_release(path, env=environ.copy())
+        summaries.append(summary)
+
+    print("\n".join(summaries))
```

Comment:
It looks like the net sum of these changes is to print on stdout what was already on stderr?
If so, all these change should be reverted.